docs/BUILD.md

```diff
@@ -298,30 +300,30 @@ The following custom packages are used:
 
 * kernel: EVE uses its own custom kernel package, rather than one of the standard linuxkit ones. This is primarily due to kernel modules and drivers, especially on arm, as well as Xen requirements.
 * `init` packages:
-    * `zededa/grub` - 
-    * `zededa/devices-trees` - 
-    * `zededa/fw` - 
-    * `zededa/xen` - 
-    * `zededa/gpt-tools` - 
-    * `zededa/dom0-ztools` - 
+    * `zededa/grub` - CoreOS inspired GRUB required to enable CoreOS-style dual partition upgrades.
+    * `zededa/devices-trees` - device trees for all the ARM platforms that EVE supports.
+    * `zededa/fw` - various firmware required for device drivers.
+    * `zededa/xen` - a single Xen binary required to boot EVE.
+    * `zededa/gpt-tools` - ChromiumOS inspired tools and sgdisk required to enable CoreOS-style dual partition upgrades. 
```

Comment:
Also, worth a deeper discussion in [UPSTREAMING.md](./UPSTREAMING.md). Be nice to get whatever capabilities are here upstreamed.